views/page/index.html

```diff
@@ -0,0 +1,48 @@
+{% extends "../component/Page.html" %}
+{% block title %}Course Management{% endblock %}
+{% block content %}
+<Page {_csrf}>
+    <div class="container container--main">
+        <div class="grid-row">
+            <div class="column-full">
+                <div class="u--space--y30 u--border--bgrey2">
+                    <div class="grid-row ">
+                        <div class="column-one-half ">
+                            <h1 class="heading heading-large heading--flush">
+                                Content Management
+                            </h1>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% for x in range(10) %}
+        <div>
+            <span>Course {{ x + 1 }}</span>
+        </div>
+        {% endfor %}
+        {% if paginatedResults.totalResults !== 0 %}
+        <div class="pager">
+            <div class="pager__controls">
+                {% if paginatedResults.currentPage > 0 %}
+                <a class="pager__prev" href="/search?q={{query}}{courseType ? '&courseType=' + courseType.split(',').join('&courseType=') : ''}&s={{searchResults.size}}&p={{searchResults.page - 1}}">Prev</a>
+                {% endif %}
```

Comment:
these href's will be changed when we actually integrate the learning catalogue api call